### PR TITLE
Add communication delay docs

### PR DESCRIPTION
**Pull request by FredericMa**
## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [x] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: [101349](https://github.com/home-assistant/core/pull/101349)
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: fixes #

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

File: source/_integrations/risco.markdown [hunk 1]
```diff
@@ -49,6 +49,8 @@ you might need to either disconnect your system from the cloud, or set up a prox
 
 The local version of the integration does not support events, and the `arming` state, but provides an additional binary sensor per zone (with the `_alarmed` suffix) that signals whether this zone is currently triggering an alarm.
 
+If the integration fails to set up you can try to increase the optional `Communication delay` parameter to wait for the specified amount of seconds between opening the connection and starting the communication with the panel.
```
Comment (frenck) [inline source/_integrations/risco.markdown]:
```suggestion
If the integration fails to set up, you can try to increase the optional `Communication delay` parameter to wait for the specified seconds between opening the connection and starting the communication with the panel.
```

File: source/_integrations/risco.markdown [hunk 2]
```diff
@@ -49,6 +49,8 @@ you might need to either disconnect your system from the cloud, or set up a prox
 
 The local version of the integration does not support events, and the `arming` state, but provides an additional binary sensor per zone (with the `_alarmed` suffix) that signals whether this zone is currently triggering an alarm.
 
+If the integration fails to set up, you can try to increase the optional `Communication delay` parameter to wait for the specified seconds between opening the connection and starting the communication with the panel.
```
Comment (frenck) [inline source/_integrations/risco.markdown]:
```suggestion
If the integration fails to set up, you can try to increase the optional **Communication delay** parameter to wait for the specified seconds between opening the connection and starting the communication with the panel.
```

**Review by frenck — DISMISSED** [review]
✅ Approved. Can be merged as soon as the parent PR gets merged.

**Comment by FredericMa** [comment]
This PR became obsolete since all happens behind the scenes now.

**Pull request closed without merging**